server/controllers/finance/reports/unpaid_invoice_payments/report.handlebars

```diff
@@ -9,47 +9,78 @@
     <strong>{{translate "REPORT.UNPAID_INVOICE_PAYMENTS_REPORT.TITLE"}}</strong>
   </h3>
 
-  <h5 style="margin:15px;" class="text-center">
+  {{#if debtorGroupName}}
+    <h4 class="text-center"><strong>{{ debtorGroupName }}</strong></h4>
+  {{/if}}
+
+  {{#if uniqueService}}
+    <h4 class="text-center"><strong>{{ uniqueService }}</strong></h4>
+  {{/if}}
+
+  <h4 style="margin:15px; font-weight:bold" class="text-center text-uppercase">
     {{date dateFrom "DD/MM/YYYY"}} - {{date dateTo "DD/MM/YYYY"}}
-  </h5>
+  </h4>
```

Comment:
If the currency is not the enterprise currency, we should show a warning that this is only temporary using this warning:

https://github.com/IMA-WorldHealth/bhima/blob/c7dd1f09205a7cd009ff86318c04630828fe1052/server/controllers/stock/reports/stock_expiration_report.handlebars#L36-L40